packages/quantic/force-app/main/default/lwc/quanticSourceCitations/quanticSourceCitations.html

```diff
@@ -0,0 +1,48 @@
+<template>
+  <template if:true={shouldDisplayCitations}>
+    <div
+      class="source-citations slds-grid slds-align_vertical-center slds-col_bump-left"
+    >
+      <p>{labels.learnMore}</p>
+      <ol
+        class="citations-container slds-list_horizontal slds-wrap slds-m-left_x-small"
+      >
+        <template
+          for:each={indexedCitations}
+          for:item="indexedCitation"
+          for:index="index"
+        >
+          <li
+            key={indexedCitation.id}
+            data-key={indexedCitation.id}
+            class="citation slds-m-around_xx-small"
+            onclick={handleCitationClick}
+          >
+            <span
+              class="slds-badge slds-badge_lightest slds-align_absolute-center"
+            >
+              <span class="slds-align_absolute-center">
```

Comment:
This was because in the documentation for badges, it mentions nested elements of a badge should be either strong, em or span.

> Along with icons, you can have nested textual elements, such as strong, em, and span elements, within a badge. Only inline and inline-block elements are supported at this time.
https://www.lightningdesignsystem.com/components/badges/#Badges-with-Nested-Elements 

Yet, it seems to still work without the span as child nested element... weird :s I can remove the span then